Sources/App/entrypoint.swift

```diff
@@ -1,21 +1,25 @@
 import Vapor
 import Logging
+import NIOCore
+import NIOPosix
 
 @main
 enum Entrypoint {
     static func main() async throws {
+        _ = NIOSingletons.unsafeTryInstallSingletonPosixEventLoopGroupAsConcurrencyGlobalExecutor()
```

Comment:
@weissi I'm not sure what the return type specifically means here and whether we need to handle it. It looks like it reports success

Reply:
sorry, fixed https://github.com/apple/swift-nio/pull/2721/files

Reply:
and yes, it reports success (or not). It checks a load of conditions (Swift versions, if some other software has also tried the takeover manoeuvre, ...)